src/content/docs/browser/single-page-app-monitoring/get-started/spa-v2.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: Browser SPA monitoring 2.0
```

Comment:
```suggestion
title: Soft navigation support for browser SPA monitoring
```

Reply:
@cwli24  What do you think of this title?

Reply:
We technically already support "soft navigations" with the current spa functionality.